chpl/chpl-service/cleanupXml.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+(set -o igncr) 2>/dev/null && set -o igncr; # this comment is required to trick cygwin into dealing with windows vs. linux EOL characters
```

Comment:
I think you should comment this line out for the final push. It's going to be run on a Linux machine, and I think we've commented out those lines on the various scripts that live on the Linux machines